sc/social/like/interfaces.py

```diff
@@ -212,3 +212,16 @@ class ISocialLikeSettings(model.Schema):
         required=False,
         default='',
     )
+
+    model.fieldset(
+        'open_graph', label=u'Open Graph', fields=['facebook_prefetch_enable'])
```

Comment:
this is Facebook related and must be on Facebook's tab.

Reply:
Ok